ultimarc/devices/usb_button.py

```diff
@@ -50,16 +48,16 @@ def set_color(self, red, green, blue):
                 raise ValueError(_('Color argument value is invalid'))
 
         data = USBButtonColorStruct(0x01, red, green, blue)
-        return self.write(USBB_SET_COLOR, USBButtonReportID, USBButtonWIndex, data, ct.sizeof(data))
+        return self.write(USBRequestCode.SET_CONFIGURATION, 0x0, USBButtonWIndex, data, ct.sizeof(data))
 
     def get_color(self):
         """
         Return the current USB button RGB color values.
         :return: (Integer, Integer, Integer) or None
         """
-        data = USBButtonColorStruct()
+        data = USBButtonColorStruct(0x01, 0x0, 0x0, 0x0)
         for x in range(20):
-            ret = self.read(USBB_GET_COLOR, USBButtonReportID, USBButtonWIndex, data, ct.sizeof(data))
+            ret = self.read(USBRequestCode.CLEAR_FEATURE, 0x0, USBButtonWIndex, data, ct.sizeof(data))
```

Comment:
Change back to USBButtonReportID for the 0x0

Reply:
Done